Client/Frontend/Intro/OnboardingCardViewController.swift

```diff
@@ -190,7 +200,7 @@ class OnboardingCardViewController: UIViewController, Themeable {
         // Adapt layout for smaller screens
         let scrollViewVerticalPadding = shouldUseSmallDeviceLayout ? UX.smallScrollViewVerticalPadding :  UX.scrollViewVerticalPadding
         let stackViewSpacingButtons = shouldUseSmallDeviceLayout ? UX.smallStackViewSpacingButtons :  UX.stackViewSpacingButtons
-        let imageViewHeight = shouldUseSmallDeviceLayout ?
+        let imageViewHeight = shouldUseTinyDeviceLayout ? UX.tinyImageViewSize.height : shouldUseSmallDeviceLayout ?
```

Comment:
nit: The logic is simple but I think is a bit long and difficult to read I wonder if a separate func makes sense for readability